apps/frontend-manage/src/components/courses/modals/CourseDuplicationModal.tsx

```
@@ -0,0 +1,735 @@
+    isGroupCreationEnabled: yup.boolean(),
+    groupCreationDeadline: initialValues?.groupDeadlineDate
+      ? yup
+          .date()
+          .required(t('manage.courseList.groupDeadlineReq'))
+          .min(
+            yup.ref('startDate'),
+            t('manage.courseList.groupDeadlineAfterStart')
+          )
+          .when('isGroupCreationEnabled', {
+            is: true,
+            then: (schema) =>
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)